client/tests/integration/upgrade.rs

```diff
@@ -3,35 +3,45 @@ use std::{path::Path, str::FromStr as _};
 use eyre::Result;
 use iroha_client::{
     client::{self, Client, QueryResult},
-    crypto::KeyPair,
     data_model::prelude::*,
 };
+use iroha_crypto::{Algorithm, KeyPair, PrivateKey};
 use iroha_logger::info;
 use serde_json::json;
 use test_network::*;
+use test_samples::ALICE_ID;
+
+const ADMIN_ID: &str =
+    "ed012076E5CA9698296AF9BE2CA45F525CB3BCFDEB7EE068BA56F973E9DD90564EF4FC@admin";
```

Comment:
ADMIN_ID needs to be agreed between the test `executor_upgrade_should_work` and the executor module `executor_with_admin`